modular_skyrat/modules/death_consequences_perk/death_consequences_trauma.dm

```diff
@@ -274,16 +268,22 @@
 		update_degradation_level()
 		update_effects()
 
+/**
+ * Updates [current_degradation_level] by comparing current degradation to max.
```

Comment:
So, as a personal preference (you don't have to feel forced to do this), I use \`this\` to highlight variables and procs in documentation strings, just because it highlights it as a proper code bit when hovered over. I think that technically the autodoc website thing parses things between [], but I don't know that anyone uses that thing anymore, I sure don't, anyway.

Reply:
got it, ill try to remember this